src/main/java/frc/robot/subsystems/climber/commands/SwitchMechanicalStopper.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.subsystems.climber.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.subsystems.climber.Climber;
+
+public class SwitchMechanicalStopper extends InstantCommand {

```

Comment:
```suggestion
public class ToggleStopper extends InstantCommand {
```
Be consistent:
1) You use "toggle" anywhere else in your code.
2) You haven't mentioned the term "mechanical stopper" anywhere in your code.

Reply:
don't apply the changes through github, refactor it in Intellij

Reply:
For some changes that can be convenient, but in general I agree.